Cargo.toml

```diff
@@ -17,6 +17,8 @@ opt-level = 3
 
 [dependencies]
 bevy = "0.14"
+# Bevy's dependencies do a lot of detailed logging, so we disable that on compile time to get a performance boost.
```

Comment:
```suggestion
# Disable low-severity logs at compile time for performance.
```

Reply:
Something similar to this is fine as well. Can just be made less wordy and explain what we're disabling (not _all_ logs).